src/config.ts

```diff
@@ -229,9 +232,14 @@ export class Config {
     fs.removeSync(this.path(p));
   }
 
-  askWriteProjectFile(p: string, content: any, force?: boolean, confirmByDefault?: boolean) {
+  async askWriteProjectFile(
+    p: string,
```

Comment:
Nit in not your code - p is a completely non descriptive variable name. Probably should rename to fileName